libs/doc-components/src/components/docTokenTable/docTokenTable.tsx

```diff
@@ -29,34 +29,51 @@ const applyStyle = (category: string, value: string, use?: string): React.CSSPro
   const style: React.CSSProperties = {};
   const categoryStyles = categoryStyleMapping[category];
 
+  if (value.endsWith('-@')) console.log(value)
+
   Object.assign(style, categoryStyles);
 
   const styleMap: { [key: string]: (value: string) => void } = {
-    'border': (val) => style.border = val,
-    'color': (val) => style.backgroundColor = val,
+    'border': (val) => style.border = `var(${val})`,
+    'color': (val) => style.backgroundColor = `var(${val})`,
     'dimension': (val) => {
       if (use === 'spacing') {
-        style.margin = val; // Example: apply margin for spacing
+        style.margin = `var(${val})`; // Example: apply margin for spacing
       } else {
-        style.borderRadius = val; // Default behavior
+        style.borderRadius = `var(${val})`; // Default behavior
       }
     },
-    'border-width': (val) => style.borderWidth = val,
-    'box-shadow': (val) => style.boxShadow = val,
-    'font-family': (val) => style.fontFamily = val,
-    'font-size': (val) => style.fontSize = val,
-    'font-weight': (val) => style.fontWeight = val,
-    'letter-spacing': (val) => style.letterSpacing = val,
-    'line-height': (val) => style.lineHeight = val,
+    'border-width': (val) => style.borderWidth = `var(${val})`,
+    'box-shadow': (val) => style.boxShadow = `var(${val})`,
+    'font-family': (val) => style.fontFamily = `var(${val})`,
+    'font-size': (val) => style.fontSize = `var(${val})`,
+    'font-weight': (val) => style.fontWeight = `var(${val})`,
+    'letter-spacing': (val) => style.letterSpacing = `var(${val})`,
+    'line-height': (val) => style.lineHeight = `var(${val})`,
+    'typography': (val) => style.font = `var(${val})`,
```

Comment:
Looks like all of these do the same string interpolation, could this be simplified somehow?